weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PromptPage/types.ts

```diff
@@ -0,0 +1,107 @@
+export type Placeholder = {
```

Comment:
Might be nice to explicitly call out that these types are OpenAI + our placeholder types